src/libraries/Microsoft.Extensions.Caching.Abstractions/src/Hybrid/HybridCacheEntryOptions.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Extensions.Caching.Distributed;
+
+namespace Microsoft.Extensions.Caching.Hybrid;
+
+/// <summary>
+/// Additional options (expiration, etc.) that apply to a <see cref="HybridCache"/> operation. When options
+/// can be specified at multiple levels (for example, globally and per-call), the values are composed; the
+/// most granular non-null value is used, with null values being inherited. If no value is specified at
+/// any level, the implementation may choose a reasonable default.
+/// </summary>
+public sealed class HybridCacheEntryOptions
+{
+    /// <summary>
+    /// Overall cache duration of this entry, passed to the backend distributed cache.
+    /// </summary>
+    public TimeSpan? Expiration { get; init; } // overall cache duration
```

Comment:
Are these comments at the ends of the lines still useful given the XML comments? If they convey something additional, seems like that should be incorporated into the XML comments.

Reply:
removed